### PR TITLE
Ie11 compat

### PR DESCRIPTION
**Pull request by cherifGsoul**
- Fix the tests for IE11

### CODE REVIEW

**Comment by matthewp** [comment]
👍 lgtm

File: can-observation_test.js [hunk 1]
```diff
@@ -527,12 +527,17 @@ QUnit.test("a bound observation with no dependencies will keep calling its funct
...
-	assert.expect(3);
+	//assert.expect(3);
 	var log = dev.log;
 	dev.log = function() {
 		dev.log = log;
-		assert.equal(arguments[0], "Observation<>", "should use can.getName");
+		// Functions in IE11 dont have name property
+		// this test is ignored under IE11
+		if (fn.name) {
+			assert.equal(arguments[0], "Observation<>", "should use can.getName");
```
Comment (justinbmeyer) [inline can-observation_test.js]: you could put expect(3) here

and an

```
else {
  assert.expect(2)
}
```
after ..

Comment (cherifGsoul) [reply can-observation_test.js]: Done!

**Comment by cherifGsoul** [comment]
@justinbmeyer This can be merged?

**Pull request merged**